add instructions for getting set up with google cloud datastore

robskrob opened the pull request.

Commit:
add instructions for getting set up with google cloud datastore

robskrob commented:
@chadwpry Here's a PR with instructions to setting up Google Cloud/Datastore. Regarding @google-cloud/datastore, it appears in [their docs](https://googlecloudplatform.github.io/google-cloud-node/#/docs/datastore/0.3.1/datastore), and I did a cursory search online for something like, `npm install --save @gcloud/datastore` but could not find an example.

chadwpry commented on README.md:
```diff
@@ -32,6 +32,60 @@ environment.
+
+    `npm install @google-cloud/datastore --save-dev`
+
+7. [Follow instructions](https://cloud.google.com/datastore/docs/) for setting up Google Cloud Datastore. After creating a project and creating two entities (Collector, Selector), you can move on.
```
I'm not sure it is necessary to create two entities at this step. Also, I think follow instructions on _this link_ is too generic. I can follow it because of using gcloud previously, but there should be more step by step instructions. I do not think there is that much involved, but listing "create a new project" is more important that pointing to google documentation.

Commit:
change npm module from `npm install gcloud --save-dev` to
`npm install --save google-cloud`, update readme to provide
the link for creating a project and creating an entity.

Commit:
update README
